conda_smithy/ci_register.py

```diff
@@ -178,48 +178,38 @@ def add_project_to_travis(user, project):
     headers = travis_headers()
     endpoint = 'https://api.travis-ci.org'
 
-    url = '{}/hooks'.format(endpoint)
+    url = '{}/{}/{}/key'.format(endpoint, user, project)
 
     found = False
     count = 0
+    response = requests.get(url, headers=headers)
+    content = response.json()
 
-    while not found:
```

Comment:
I think we were only raising it to inspect the message we got back. Before we suppressed it. Given as we now understand the issue, I think we can go back to suppressing it or checking for a failure and retrying.